pkg/controllers/applicationset_controller_test.go

```diff
@@ -57,6 +57,7 @@ func TestExtractApplications(t *testing.T) {
 		template			argoprojiov1alpha1.ApplicationSetTemplate
 		generateParamsError	error
 		rendererError		error
+		expectErr bool
```

Comment:
Per these changes we now expect these tests to do something differently, as we don't want to swallow an error, we want to return it and let the queue retry with backoff.